test/e2e/framework/replicaset/wait.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2017 The Kubernetes Authors.
+Copyright 2019 The Kubernetes Authors.
```

Comment:
Can we undo this? Moving files shouldn't touch copy right dates IIRC :)

Reply:
Seems like a git issue, `wait.go` is actually a new file which is quite different from the original `rs_util.go`. 🤣

Reply:
yes, k8s keeps the original date.

Reply:
It's a new file but I believe the date still needs to be preserved if git sees it as a move

Reply:
Updated the date in the latest commit.

Reply:
please squash the commits into one and force push, so that we have a single commit.